index.html

```diff
@@ -49,14 +35,14 @@
         <img class="robot-5" src="images/hardware-robot.svg"/>
       </div>
 	</div>
+  <div class="placeholder"></div>
   <nav>
     <img class="nav-logo" src="images/logo.png"/>
     <div class="nav-option-bar">
-      <a class="nav-option">Register</a>
+      <a href="https://gohacksc.typeform.com/to/gi49kN" class="nav-option">Register</a>
```

Comment:
You should change this to:
//gohacksc.typeform.com/to/gi49kN
Never specify what type of protocol is being used in the link.